### DISCUSSION TITLE
include all sub-directorie readme-files

### DISCUSSION THREAD

**Discussion by s-light** [vuepress/core#1590] [Q&A]
Hello Community,

i would like to create a single page that includes all the sub-directories readme.md files. (recursively)
(and add a vue based simple client local search and filter option)

is there a easy way to do this?
i know it is very different from the base concept of vuepress..

i found the [include function in the mdEnhancePlugin](https://plugin-md-enhance.vuejs.press/guide/content/include.html).
could i combine this with a classic vue [v-for](https://vuejs.org/api/built-in-directives.html#v-for) and somehow *get / list* all the sub-directories?

the last part - the *get the list of directories* is where i have no idea how to go about...

i am open for any hints and ideas -
also if you have an idea for another static-site-generator thing that can do something like this... pleas let me know!

sunny greetings
stefan

**Comment by meteorlxy** [comment]
Maybe you can try vitepress or [unplugin-vue-markdown](https://github.com/unplugin/unplugin-vue-markdown) to import md as vue component.

**Comment by s-light** [comment]
thanks meteorlxy,
vitepress has [dynamically-generating-paths](https://vitepress.dev/guide/routing#dynamically-generating-paths) - and maybe i can tweak this to do what i want...

so vuepress has no such dynamically routing feature?
